pkg/authorizer/interfaces.go

```diff
@@ -18,3 +18,13 @@ type Interface interface {
 type ACRTokenExchanger interface {
 	ExchangeACRAccessToken(armToken types.AccessToken, tenantID, acrFQDN string) (types.AccessToken, error)
 }
+
+// MIARMTokenRetriever is the interface to retrieve an ARM access token via managed identity.
+type MIARMTokenRetriever interface {
+	MIAcquireARMToken(clientID, resourceID string) (types.AccessToken, error)
```

Comment:
Could we just call this method `AcquireARMToken`? It's ok for both the MI and WI token retrievers to have the same method name for their interfaces